zerver/lib/embedded_docs.py

```diff
@@ -0,0 +1,22 @@
+from django.template import loader
+import os
+
+from zerver.models import EmbeddedDocArticle
+
+def import_markdown_directory(directory: str) -> None:
+    """
+    Imports and saves a Markdown directory into a database.
+    Useful for setting up Postgres-based search of those directories.
+    """
+    articles = []
+    for entry in os.listdir("templates/" + directory):
```

Comment:
it would be nice if we could accept something of the form of MarkdownDirectoryView.path_template, just for consistency's sake.

Reply:
I can't figure out a cleaner way of doing this, so I just further documented what `directory` really is, and left it as-is. If there are other ways of dealing with this, I would love to hear about them.